controllers/Handle.php

```diff
@@ -26,7 +26,7 @@ class Handle extends Controller
     /**
      * Defines a collection of actions available without authentication.
      */
-    protected array $publicActions = [
+    protected $publicActions = [
```

Comment:
What was the reason for this change?

Reply:
![image](https://github.com/wintercms/wn-sso-plugin/assets/12476942/5d74e9ab-1a79-4932-ba80-d9b10754de1c)

This variable must be typed in the controller from which it is extended i think